DonjinKrawler-Client/src/main/java/donjinkrawler/items/ItemMaker.java

```diff
@@ -1,21 +1,43 @@
 package donjinkrawler.items;
 
+import donjinkrawler.items.tiers.CommonTier;
+import donjinkrawler.items.tiers.LegendaryTier;
+import donjinkrawler.items.tiers.RareTier;
+import donjinkrawler.items.tiers.Tier;
 import krawlercommon.items.*;
 
+import java.util.Random;
+
 public class ItemMaker {
 
     public static BaseItem makeItem(ItemLocationData itemData) {
         if (itemData instanceof ArmorData) {
-            return new Armor((ArmorData) itemData);
+            return new Armor((ArmorData) itemData, getTier(), 10, 5, 5);
```

Comment:
No need to cast if the declared type in the constructor is the needed one, not sure why I added the casting but you can remove it ;)

Reply:
Oky doky

Reply:
Nekompiliuoja

Reply:
Paliksiu kaip yra.